dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/FeatureResolverWrapper.java

```diff
@@ -318,6 +325,28 @@ public static RequiredFeatureMap getAllRequiredFeatures(WebSphereRuntime wr, Lis
                 fe.getRequiredFeatures(wr, supportedModules, supportedDeltas, existingFeatures, featureMap, includeAll, monitor);
             }
         }
+
```

Comment:
Should this code be called from getAllRequiredFeaturesBasedOnFacets as well?

Reply:
Sure. I'll refactor out into a helper method and call in both places.